src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -468,7 +476,7 @@ public int getMaxChunksInBlob() {
     Object val =
         this.parameterMap.getOrDefault(
             MAX_CHUNKS_IN_BLOB,
-            isIcebergMode ? MAX_CHUNKS_IN_BLOB_ICEBERG_MODE_DEFAULT : MAX_CHUNKS_IN_BLOB_DEFAULT);
+            isIcebergMode() ? MAX_CHUNKS_IN_BLOB_ICEBERG_MODE_DEFAULT : MAX_CHUNKS_IN_BLOB_DEFAULT);
```

Comment:
same comment as above, this defaulting was already done, why do all these methods do it again?